src/components/organisms/SfBottomNavigation/SfBottomNavigation.scss

```diff
@@ -1,37 +1,52 @@
-@import '../../../css/variables';
+@import "../../../css/variables";
+
+$bottom-navigation-cart-shadow: 0px 4px 6px rgba(66, 66, 66, 0.2) !default;
```

Comment:
it should be `__cart` and not curt but just `__floating-btn`

Reply:
I've deleted it, it's in SfCircleButton now.